### ISSUE TITLE
App crashes on start when another VPN is running

### ISSUE THREAD

**Issue by larsks** [pacien/tincapp#44]
I have created a tinc configuration in `/storage/emulated/0/Android/data/org.pacien.tincapp/files/vpn/`.  `tinc.conf` looks like:

```
Name = oneplus
ConnectTo = outside
```

And network.conf looks like:

```
Address = 172.16.254.50/32
Route = 172.16.254.0/24
```

And `hosts/outside` looks like (-ish):

```
Address = tinc.example.com

-----BEGIN RSA PUBLIC KEY-----
...
-----END RSA PUBLIC KEY-----
```

As soon as I select a network name in the UI, the tinc app crashes.  There are no logs in `/storage/emulated/0/Android/data/org.pacien.tincapp/cache`.

I am running Android `7.1.2` (lineageos `14.1-<a recent update>`).

**Comment by pacien** [comment]
Hello,
What is the error message shown?

**Comment by larsks** [comment]
There is no particular error message.  Tinc crashes and you get the standard "The application tinc has crashed..." dialog that lets you (a) reopen the app or (b) mute the error.

But in any case I figured out the problem: this appears to be tinc's way of saying "Hey, there is currently another VPN configured". I run [blokada](https://blokada.org/index.html), which implements ad blocking as a vpn service.

It would be nice if tinc would display an error ("can't run because a vpn is already active") rather than crashing.

Once I stopped Blokada, I was able to get tinc running without a problem.
